plugins/legacy-preset-chart-big-number/test/transformProps.test.ts

```diff
@@ -135,7 +135,7 @@ describe('BigNumber', () => {
         },
       };
       const transformed = transformProps(propsWithDatasource);
-      expect(transformed.formatNumber(transformed.bigNumber)).toStrictEqual('1.23');
+      expect(transformed.headerFormatter(transformed.bigNumber)).toStrictEqual('1.23');
```

Comment:
Can you add a test for timestamp?

Reply:
I will push a separate PR after this. Thanks!